x/margin/keeper/migrations.go

```diff
@@ -0,0 +1,20 @@
+package keeper
```

Comment:
This file needs build tags, it's breaking the tests @canercandan

Reply:
thanks, added the build tags to the file